spyder/config/main.py

```diff
@@ -467,11 +467,21 @@
               }),
             ('color_schemes',

```

Comment:
Please rename this section to `appearance`, remove the options you moved from the `general` section to here and bump `CONF_VERSION`.